sdk/src/types/block/address/bech32.rs

```diff
@@ -18,12 +18,21 @@ use packable::{
 
 use crate::types::block::{address::Address, ConvertTo, Error};
 
-#[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash)]
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
 pub struct Hrp {
     inner: [u8; 83],
     len: u8,
 }
 
+impl core::fmt::Debug for Hrp {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        f.debug_struct("Hrp")
+            .field("inner", &prefix_hex::encode(&self.inner[..self.len as usize]))
```

Comment:
For the record, it was previously also not helpful

Reply:
How is it not helpful?

Reply:
That's what the display impl is for. Debug is for debugging

Reply:
Yeah sure, but there are no practical use of HRP as raw bytes. If anything, I'd say it's preventing us from debugging correctly as we'd have to manually convert it to see what's up

Reply:
You could also argue that having both impls print the string could cause debugging issues because you need to convert back to bytes. I like that if you want the string, you can just use display.

Reply:
Sure but you can't do anything when it's nested in something and you are using debug 🤷🏻‍♂️